src/otx/core/exporter/visual_prompting.py

```diff
@@ -93,12 +95,17 @@ def to_openvino(
                 tmp_dir,
                 base_model_name,
                 OTXPrecisionType.FP32,
-                False,
+                embed_metadata=False,
             )
             exported_model = openvino.convert_model(tmp_dir / (base_model_name + ".onnx"))
 
         exported_model = self._postprocess_openvino_model(exported_model)
 
+        if self.metadata is not None:
```

Comment:
It seems that this part can be executed in `self._postprocess_openvino_model` if you update `src/otx/core/types/export.py`. How about doing that to reduce duplicated code?

Reply:
No, it can not, unless we introduce more global changes to exporter architecture. 
We have two models here, and therefore two difference `model_type` parameters should be written.
Generalizing that, we should extend export to the case of multiple IRs per model, each of those could require an individual set of export parameters. Also, this is the only why we have a standalone exporter for visual prompting.

Reply:
For now, I'd like to keep the core exporter clean, while making workarounds as explicit as possible, at the cost of some duplication though.
Here we can also temporally mock `self.metadata`, but as I said let's keep workarounds explicit.